src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Controls.Sdk.targets

```diff
@@ -0,0 +1,18 @@
+<Project>
+
+  <PropertyGroup>
+    <UsingMicrosoftMauiControlsSdk>true</UsingMicrosoftMauiControlsSdk>
+  </PropertyGroup>
```

Comment:
Split controls into a separate targets pair because we need a few things for UseMauiCore only, but some things are Controls only:
 - set defaults for EnableDefaultMaui*
 - remove platform usings
 - import xaml and single project things